### PR TITLE
Replace SIG/WG meeting placeholder references with Zoom links-5

### LINKED ISSUE #4588: Replace SIG/WG meeting placeholder references with Zoom links
Some time ago Zoom did not have sufficient moderation capabilities and the Zoom meeting links were removed from [sigs.yaml] and pointed towards this placeholder:
https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit

As a workaround to help mitigate trolls from causing problems in meetings. This has since been resolved and the entries in [sigs.yaml] should be updated with their correct link.

There are 53 references to the above placeholder as of 2020-03-09. Updated zoom links can be taken from either the SIG/WG's agenda or their calendar event from the [community calendar](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles).

/area community-management
/sig contributor-experience
/help
/good-first-issue

[sigs.yaml]: https://github.com/kubernetes/community/blob/master/sigs.yaml

### PR DESCRIPTION
**Pull request by AGMETEOR**
**Which issue(s) this PR fixes**:
Fixes #4588

### CODE REVIEW

**Comment by AGMETEOR** [comment]
Some notes
------------
* Cluster Federation (KubeFed) Working Group - Zoom link for “7:30am PST” is not available

* Regular SIG Meeting - Zoom link for “9:30am PST” is not available

**Comment by mrbobbytables** [comment]
huh..thats weird, my other comments in the review didn't post...will follow up again.

File: sigs.yaml [hunk 1]
```diff
@@ -1425,15 +1425,15 @@ sigs:
     time: "7:30"
     tz: PT (Pacific Time)
     frequency: weekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```
Comment (mrbobbytables) [inline sigs.yaml]:
```suggestion
    url: https://zoom.us/my/k8s.mc
```

ref: https://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY/edit#

File: sigs.yaml [hunk 2]
```diff
@@ -1425,15 +1425,15 @@ sigs:
...
     archive_url: https://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY/edit
     recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP3iKP5EzMbtNT2zOZv6RCrX
   - description: Regular SIG Meeting
     day: Tuesday
     time: "9:30"
     tz: PT (Pacific Time)
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```
Comment (mrbobbytables) [inline sigs.yaml]:
```suggestion
    url: https://zoom.us/my/k8s.mc
```
ref: https://docs.google.com/document/d/18mk62nOXE_MCSSnb4yJD_8UadtzJrYyJxFwbrgabHe8/edit

**Comment by mrbobbytables** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**